pa11y.json

```diff
@@ -0,0 +1,3 @@
+{
+  "hideElements": "iframe"
```

Comment:
From pa11y doc on hideElement (https://github.com/pa11y/pa11y#hideelements-string): "A CSS selector to hide elements from testing". What is the iframe on the application that we don't want to test?